example/WORKSPACE.bazel

```diff
@@ -239,6 +239,10 @@ load("@aspect_rules_lint//lint:vale.bzl", "fetch_vale")
 
 fetch_vale()
 
+load("@aspect_rules_lint//lint:ktlint.bzl", "fetch_ktlint")
```

Comment:
this file needs to install rules_kotlin as well, currently the CI fails when testing the example under `--noenable_bzlmod`

Reply:
Updated to include `rules_kotlin` there. I also had to explicitly declare a newer version of rules_java because i was getting this 
```
ERROR: Traceback (most recent call last):
        File "/home/owner/.cache/bazel/_bazel_owner/9bb43054d613384342b45fabf618b109/external/rules_kotlin/third_party/jarjar.bzl", line 14, column 37, in <toplevel>
                load("@rules_java//java:defs.bzl", "JavaInfo")
```
with rules_kotlin 1.9.0 (which seems to correspond to rules_java 6.4.0 per bcr)

Reply:
yeah I also ran into that error trying to fix it. Good thing WORKSPACE will go away!